<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshFromRaster</class>
 <widget class="QDialog" name="MeshFromRaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create mesh from raster...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="interpretationGroupBox">
     <property name="title">
      <string>Interpret intensities as</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="elevationButton">
        <property name="text">
         <string>Elevation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="materialButton">
        <property name="text">
         <string>Materials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ignoreButton">
        <property name="text">
         <string>Ignore</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="elementTypeGroupBox">
     <property name="title">
      <string>Represent pixels as</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="triButton">
        <property name="text">
         <string>sets of two triangles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="quadButton">
        <property name="text">
         <string>rectangles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hexButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>hexahedra</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mshNameLabel">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>    Name of mesh:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mshNameEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeshFromRaster</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeshFromRaster</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
